test/system/group_offer_categories_test.rb

```diff
@@ -0,0 +1,37 @@
+require 'application_system_test_case'
+
+class GroupOfferCategoriesTest < ApplicationSystemTestCase
+  def setup
+    @user = create :user
+    login_as @user
+  end
+
+
+  test 'create new group offer category and update it' do
```

Comment:
In general, each test should test only one thing. So please slip that test in two.

Reply:
This is true for unit tests but for system tests it's sometimes easier to test a whole workflow, though it can also make it trickier to update the test in the future.

In general I can recommend writing more unit tests where you test all edge cases (they're simpler to write and run a lot faster), and using system tests for general workflows and things you can't easily test directly (e.g. JS interactions).